code/modules/mob/living/carbon/superior_animal/roach/blattedin.dm

```diff
@@ -0,0 +1,10 @@
+/datum/reagent/toxin/blattedin/touch_mob(var/mob/living/carbon/superior_animal/roach/bug, amount)
+	if(istype(bug))
+		if(bug.stat == DEAD)
+			if((bug.blattedin_revives_left >= 0) || prob(70))//Roaches sometimes can come back to life from healing vapors
```

Comment:
Nobody really got to test it before since it was broken, but these values are pretty silly as is

Change that || to && so that it's hard capped

Reply:
I didn't even notice that, will fix.